extensions/wikia/Flags/controllers/templates/FlagsController_editForm.php

```diff
@@ -8,13 +8,20 @@
 			<label for="<?= Sanitizer::encodeAttribute( "{$prefix}:{$inputNameCheckbox}" ) ?>"><?= $flag['flag_name'] ?></label>
 			<a href="<?= Sanitizer::cleanUrl( $flag['flag_view_url'] ) ?>" target="_blank"><?= wfMessage( 'flags-edit-form-more-info' )->escaped() ?></a>
 			<?php
-				$flagParamsNames = json_decode( $flag['flag_params_names'] );
+			$flagParamsNames = json_decode( $flag['flag_params_names'] );
+			if ( !empty( $flagParamsNames ) ): ?>
+				<fieldset class="params">
+				<?php
 				foreach ( $flagParamsNames as $flagParamName => $flagParamDescription ):
 					$flagParamValue = (string)$flag['params'][$flagParamName];
-			?>
-			<input type="text" name="<?= Sanitizer::encodeAttribute( "{$prefix}:{$flagParamName}" ) ?>" value="<?= Sanitizer::encodeAttribute( $flagParamValue ) ?>" placeholder="<?= Sanitizer::encodeAttribute( $flagParamDescription ) ?>" class="param">
-			<?php
+					?>
```

Comment:
IMHO it's better to leave the `?>` tag in the same column that the opening tag. What do you think?